index.jsx

```diff
@@ -0,0 +1,94 @@
+import { css } from 'uebersicht';
+
+// Root node CSS
+export const className = css`
+  min-width: 150px;
+
+  top:    1em;
+  bottom: 0;
+  right:  3%;
+
+  font-family:     'SFNS Display', 'Helvetica Neue', sans-serif;
+  font-smoothing:  antialiased;
+  color:           white;
+  font-size:       32px;
+  font-weight:     bold;
+  letter-spacing:  0.025em;
+  line-height:     .9em;
+
+  text-align:      right;
+  text-transform:  uppercase;
+
+  opacity: 0.5;
+`;
+
+const iconCss = css`
+  background: url('/hddspace.widget/PNG/hdd-32.png');
+  -webkit-filter: drop-shadow(1px 1px 0 rgba(0, 0, 0, .4));
+  filter: drop-shadow(1px 1px 0 rgba(0, 0, 0, .4));
+`;
+
+const bgCss = css`
+  height: 40px;
+  width: 50px;
+  background-position: center center;
+  background-repeat: no-repeat;
+  background-size: 32px 32px;
+  margin-right: 8px;
+`;
+
+const diskCapacityCss = css`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: flex-end;
+`;
+
+const diskRowCss = css`
+  margin-bottom: 32px;
+`;
+
+const nameCss = css`
+  font-size: 11px;
+  line-height: 18px;
+`;
+
+export const command = `{ df -Hl; diskutil list; } | grep "disk\\ds"`;
```

Comment:
This command shows only local drives, and the `diskutil` part is used so we can find the real drive partition names